efficientdet/main.py

```diff
@@ -396,5 +392,6 @@ def terminate_eval():
 
 
 if __name__ == '__main__':
+  logging.set_verbosity(logging.WARNING)
   tf.disable_eager_execution()
-  tf.app.run(main)
+  app.run(main)
```

Comment:
I changed from tf.app.run to app.run because app.run would automatically check whether flags are valid or not.

Reply:
nice!